### ISSUE TITLE
REST api for Compose

### ISSUE THREAD

**Issue by thbeh** [docker/compose#230]
Hi, is there a REST api for fig in the pipeline?

Thanks
Regards
Beh

**Comment by schickling** [comment]
+1

**Comment by dnephin** [comment]
Some related issues #1033, #954

**Comment by aramalipoor** [comment]
:+1:

**Comment by cshg** [comment]
👍

**Comment by meoooh** [comment]
+1

**Comment by nskforward** [comment]
+1

**Comment by egulatee** [comment]
+1

Use case:  Jenkins running in a container could use docker-compose REST URL to restart "composed" containers once the build is complete.

**Comment by sjahreis** [comment]
+1

Use case: A gradle plugin which deploys your applications automatically.

**Comment by rootsongjc** [comment]
+1

**Comment by philippelabat** [comment]
+1

**Comment by Cerfoglg** [comment]
+1

I'm currently working on a tool to deploy and manage a series of services on remote Docker Swarm clusters, so having the ability to call Docker Compose via a REST API would be incredibly useful.

**Comment by mrsrinivas** [comment]
+1

**Comment by monokal** [comment]
+1

**Comment by renatoathaydes** [comment]
+1

I wanted to be able to have automated tests that can simulate other services going down. Using a docker-compose API, that would be much easier.

**Comment by shin-** [comment]
Closing as `wontfix` for reasons I explained here: https://github.com/docker/compose/issues/5545#issuecomment-356100870
